server/public/styles.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.navbar-brand {
+  margin-left: 165px;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
```

Comment:
Instead of modifying Bootstrap's classes, make your own and use them with the Bootstrap classes.